database/fake_data.sql

```diff
@@ -0,0 +1,295 @@
+('2024-10-08 00:10:25', 3),
+('2024-10-08 01:25:40', 2),
+('2024-10-08 02:40:55', 1),
+('2024-10-08 03:55:10', 1),
+('2024-09-30 14:20:35', 2),
+('2024-10-01 08:45:20', 1),
+('2024-10-02 19:30:50', 3),
+('2024-10-03 11:55:15', 1),
+('2024-10-04 16:10:40', 2),
+('2024-10-05 09:25:30', 1),
+('2024-10-06 13:50:45', 3),
+('2024-10-06 18:05:55', 1),
+('2024-10-07 07:40:10', 2),
+('2024-10-07 22:15:25', 1);
+
+
+INSERT INTO forecast (county_id, temperature_c, precipitation_probability_percent, precipitation_mm, cloud_coverage_percent, visibility_m, at)
+VALUES
+(12, 18.5, 30, 1.1, 20, 10000, '2024-09-30 14:20:35'),
+(45, 15.2, 60, 2.5, 85, 8000, '2024-10-01 09:30:45'),
+(8, 21.1, 10, 1.1, 10, 15000, '2024-10-02 18:15:20'),
+(91, 13.8, 70, 3.2, 90, 5000, '2024-10-03 12:45:55'),
+(67, 17.6, 40, 1.5, 60, 12000, '2024-10-04 16:25:10'),
+(23, 20.1, 20, 1.1, 25, 14000, '2024-10-05 08:50:05'),
+(58, 14.3, 80, 4.8, 95, 6000, '2024-10-06 14:55:35'),
+(30, 19.4, 50, 1.2, 45, 11000, '2024-10-06 19:20:50'),
+(7, 22.1, 15, 1.1, 12, 16000, '2024-10-07 07:10:25'),
+(83, 16.5, 65, 2.9, 80, 9000, '2024-10-07 22:35:40');
+
+
+INSERT INTO image (image_name, image_url, image_date, region_id, constellation_id)
+VALUES
+('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-01', 1, 1),
+('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-02', 2, 2),
+('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-03', 3, 3),
+('ASTRO_moon', 'https://upload.wikimedia.org/wikipedia/commons/thumb/e/e1/FullMoon2010.jpg/1200px-FullMoon2010.jpg', '2024-10-04', 4, 4),
+('ASTRO_moon', 'https://upload.wikimedia.org/wikipedia/commons/thumb/e/e1/FullMoon2010.jpg/1200px-FullMoon2010.jpg', '2024-10-05', 5, 5),
+('ASTRO_star', 'https://dq0hsqwjhea1.cloudfront.net/Interactive-Sky-Chart-600px.jpg', '2024-10-06', 6, 6),
+('ASTRO_star', 'https://dq0hsqwjhea1.cloudfront.net/Interactive-Sky-Chart-600px.jpg', '2024-10-07', 7, 7),
+('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-08', 8, 8),
+('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-09', 9, 9),
+('ASTRO_moon', 'https://upload.wikimedia.org/wikipedia/commons/thumb/e/e1/FullMoon2010.jpg/1200px-FullMoon2010.jpg', '2024-10-10', 10, 10),
+('ASTRO_star', 'https://dq0hsqwjhea1.cloudfront.net/Interactive-Sky-Chart-600px.jpg', '2024-10-11', 11, 11),
+('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-12', 12, 12),
+('ASTRO_moon', 'https://upload.wikimedia.org/wikipedia/commons/thumb/e/e1/FullMoon2010.jpg/1200px-FullMoon2010.jpg', '2024-10-13', 13, 13),
+('ASTRO_star', 'https://dq0hsqwjhea1.cloudfront.net/Interactive-Sky-Chart-600px.jpg', '2024-10-14', 14, 14),
+('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-15', 15, 15),
```

Comment:
Could some of these just have NULL values for region and constellation?

Reply:
I believe in the schema the constrains prevent nulls

Reply:
Would be good to test the extended functionality of our ERD

Reply:
Can confirm we can have nulls here, as the schema specifically allows it for region ID and constellation ID.